docs/en/mavlink/index.md

```diff
@@ -0,0 +1,88 @@
+# MAVLink Messaging
+
+[MAVLink](https://mavlink.io/en/) is a very lightweight messaging protocol that has been designed for the drone ecosystem.
+
+PX4 uses _MAVLink_ to communicate with ground stations and MAVLink SDKs, such as _QGroundControl_ and [MAVSDK](https://mavsdk.mavlink.io/), and as the integration mechanism for connecting to drone components outside of the flight controller: companion computers, MAVLink enabled cameras, and so on.
+
+This topic provides a brief overview of fundamental MAVLink concepts, such as messages, commands, and microservices.
+It also links instructions for how you can add PX4 support for:
+
+- [Adding Standard Messages](../mavlink/adding_messages.md)
+- [Streaming MAVLink messages](../mavlink/streaming_messages.md)
+- [Handling incoming MAVLink messages (and writing to a uORB topic)](../mavlink/receiving_messages.md)
+- [Custom MAVLink Messages](../mavlink/custom_messages.md)
+- [Protocols/Microservices](../mavlink/protocols.md)
+
+::: info
+We do not yet cover _command_ handling and sending, or how to implement your own microservices.
+:::
+
+## MAVLink Overview
+
+MAVLink is a lightweight protocol that was designed for efficiently sending messages over unreliable low-bandwidth radio links.
+
+_Messages_ are simplest and most "fundamental" definition in MAVLink, consisting of a name (e.g. [ATTITUDE](https://mavlink.io/en/messages/common.html#ATTITUDE)), id, and fields containing relevant data.
+They are deliberately lightweight, with a constrained size, and no semantics for resending and acknowledgement.
+Stand-alone messages are commonly used for streaming telemetry or status information, and for sending commands where no acknowledgement is required - such as setpoint commands sent at high rate.
+
+The [Command Protocol](https://mavlink.io/en/services/command.html) is a higher level protocol for sending commands that may need acknowledgement.
+Specific commands are defined as values of the [MAV_CMD](https://mavlink.io/en/messages/common.html#mav_commands) enumeration, such as the takeoff command [MAV_CMD_NAV_TAKEOFF](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_TAKEOFF), and include up to 7 numeric "param" values.
+The protocol sends a command by packaging the parameter values in a `COMMAND_INT` or `COMMAND_LONG` message, and waits for an acknowledgement with a result in a `COMMAND_ACK`.
+The command is resent automatically if no acknowledgment is received.
+Note that [MAV_CMD](https://mavlink.io/en/messages/common.html#mav_commands) definitions are also used to define mission actions, and that not all definitions are supported for use in commands/missions on PX4.
+
+[Microservices](https://mavlink.io/en/services/) are other higher level protocols built on top of MAVLink messages.
```

Comment:
I would have put commands under microservices as well, but I would call it "meta protocols" at least in brackets.

And you could add parameters.